flo-runner/src/main/java/com/spotify/flo/context/FloRunner.java

```diff
@@ -81,6 +121,10 @@ private static Config defaultConfig() {
    * @return a {@link Result} with the value and throwable (if thrown)
    */
   public static <T> Result<T> runTask(Task<T> task, Config config) {
+    if (config.getBoolean("termination.logging")) {
```

Comment:
We could make this method non-static. Would that be easier for the internal part?